packages/core-components/src/components/button/button.e2e.ts

```diff
@@ -0,0 +1,49 @@
+import { setupComponent, setupComponentType, performAccessibilityCheck } from '../../utils/test-utils';
+
+async function setup(html: string) : setupComponentType {
+  return await setupComponent(html, 'o-button');
+}
+
+describe('o-button', () => {
```

Comment:
All `o-` prefixes should be changed to `ods-` to maintain consistency with the system